de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/AbstractOutgoingReferencePointNegotiation.java

```diff
@@ -56,10 +56,10 @@
 
     private PacketCollector startActivityQueuingResponseCollector;
 
-    protected AbstractOutgoingProjectNegotiation( //
+    protected AbstractOutgoingReferencePointNegotiation( //
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Constructor has 11 parameters, which is greater than 7 authorized. ([squid:S00107](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS00107))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWaWrfcSsSPV6ZXI_B9O&open=AWaWrfcSsSPV6ZXI_B9O)

[ISSUE_KEY]: AWaWrfcSsSPV6ZXI_B9O